e2e/nomostest/webhook.go

```diff
@@ -44,39 +43,29 @@ func StopWebhook(nt *NT) {
 		return
 	}
 	webhookName := configuration.Name
-	webhookGK := "validatingwebhookconfigurations.admissionregistration.k8s.io"
+	webhookGK := "ValidatingWebhookConfiguration"
 
-	out, err := nt.Shell.Kubectl("annotate", webhookGK, webhookName, fmt.Sprintf("%s=%s", metadata.WebhookconfigurationKey, metadata.WebhookConfigurationUpdateDisabled))
-	if err != nil {
-		nt.T.Fatalf("got `kubectl annotate %s %s %s=%s` error %v %s, want return nil",
-			webhookGK, webhookName, metadata.WebhookconfigurationKey, metadata.WebhookConfigurationUpdateDisabled, err, out)
-	}
-
-	err = nt.Watcher.WatchObject(kinds.ValidatingWebhookConfiguration(),
-		webhookName, "",
-		[]testpredicates.Predicate{
-			testpredicates.HasAnnotation(metadata.WebhookconfigurationKey, metadata.WebhookConfigurationUpdateDisabled),
-		},
-		testwatcher.WatchTimeout(30*time.Second))
-	if err != nil {
-		nt.T.Fatal(err)
+	err := nt.KubeClient.Get(webhookName, "", &admissionv1.ValidatingWebhookConfiguration{})
```

Comment:
No need to Get before Delete. You can delete the webhook config, and then check if the error is NOT_FOUND.

Reply:
Done